earth_enterprise/rpms/build.gradle

```diff
@@ -43,6 +56,29 @@ apply plugin: "nebula.ospackage"
 
 import org.opengee.shell.GeeCommandLine
 
+// Varables used for specifying location of two files
+// 'gee_long_version.txt' and 'gepgcommon'
+def src_dir = "../src/"
+def scons_option = "release=1"
+
+// Look for command line paramters that specify how the packages are
+// to be built.  This can be either 'optimize' or 'debug'.
+// Option 'release' is the default.
+def native_dir = {
+  if ( project.hasProperty( 'optimize' ) ) {
+    scons_option = "optimize=1"
+    return "NATIVE-OPT-x86_64"
+  }  // end check for option 'optimze'
+  if ( project.hasProperty( 'debug' ) ) {
+    scons_option = ""
```

Comment:
This doesn't work.  Passing an empty command-line argument to `scons` causes a failure:

```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/pav/prod/bitbucket.com/t-sciences/geoplatform/earthenterprise_code/earthenterprise/earth_enterprise/rpms/build.gradle' line: 111

* What went wrong:
A problem occurred evaluating root project 'rpms'.
>                 Runing `scons` to produce <../src/NATIVE-DBG-x86_64/gee_long_version.txt> failed!
                  Command: [/usr/bin/scons, , ../src/NATIVE-DBG-x86_64/gee_long_version.txt]
                  Exit code: 2.
                  Error output:
                  
  scons: warning: The build_dir keyword has been deprecated; use the variant_dir keyword instead.
  File "/home/pav/prod/bitbucket.com/t-sciences/geoplatform/earthenterprise_code/earthenterprise/earth_enterprise/src/SConstruct", line 519, in <module>
  fusion/mtjtools/analyze/analyze.cpp:21:23: fatal error: gdal_priv.h: No such file or directory
  compilation terminated.
  scons: *** [NATIVE-DBG-x86_64/fusion/mtjtools/analyze/analyze.o] Error 1
  
                  Standard output:
                  scons: Reading SConscript files ...
  Checking for C library gtest... no
  Checking for C header file libpng12/png.h... yes
  scons: done reading SConscript files.
  scons: Building targets ...
  scons: building associated VariantDir targets: NATIVE-DBG-x86_64
  g++ -o NATIVE-DBG-x86_64/fusion/mtjtools/analyze/analyze.o -c -g -pipe -pthread -Wall -Werror -Wno-parentheses -Wno-error=unused-result -Wno-error=unused-local-typedefs -Wno-error=unused-but-set-variable -I/usr/include/libpng12 -D_LARGEFILE64_SOURCE -DQT_THREAD_SUPPORT -DQT_NO_ASCII_CAST -DQT_CLEAN_NAMESPACE -D_REENTRANT -DGUNIT_NO_GOOGLE3 -DKH_MAX_NUM_JOBS_COEFF=16 -INATIVE-DBG-x86_64/include -INATIVE-DBG-x86_64 -I. -INATIVE-DBG-x86_64/common -Icommon -Icommon/khmisc -Ikeyhole -INATIVE-DBG-x86_64/fusion -Ifusion -Ifusion/mtjtools/histogram fusion/mtjtools/analyze/analyze.cpp
  scons: building terminated because of errors.
```

You can set `scons_option` to `release=0`, or make this variable an array that gets combined with the other elements to construct the `scons` command line.